src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const listTree = document.querySelector('.tree');
+const list = document.querySelectorAll('li');
+
+for (const el of list) {
+  const createSpan = document.createElement('span');
```

Comment:
```suggestion
  const newSpan = document.createElement('span');
```
Don't name it like a function. Function names usually start with imperative verbs